lib/checks/aria/unsupportedrole.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "unsupportedrole",
+  "evaluate": "unsupportedrole.js",
+  "metadata": {
+    "impact": "critical",
+    "messages": {
+      "pass": "ARIA role is supported",
+      "fail": "The role used is not widely supported in assistive technologies"
```

Comment:
Should this say "in screen readers", or "in screen readers and other assistive technologies"?